docs/debugging.md

```diff
@@ -0,0 +1,59 @@
+---
+id: debugging
+title: Debugging
+---
+
+Resolve uses the [resolve-debug-levels](https://www.npmjs.com/package/resolve-debug-levels) package to log debug information. This package extends the [debug](https://www.npmjs.com/package/debug) library's functionality with logging levels. The package provides the following logging levels:
+
+| Level   | Description                                            |
+| ------- | ------------------------------------------------------ |
+| log     | Messages that should always be displayed               |
+| error   | Errors that can hinder normal program execution        |
+| warn    | Potential problems in the application's implementation |
+| debug   | Information displayed for debugging purposes           |
+| info    | Messages that describe the current operation           |
+| verbose | Extended descriptions to the previous message          |
```

Comment:
"to the previous message" Why 'to'?

Reply:
Changed to 'Extended descriptions for the previous message'

Reply:
Ok